### PR TITLE
feat (docs): [streamline] readme instructions

### PR DESCRIPTION
**Pull request by ferantivero**
closed: #516483

## WHY

as part of the process of testing the Fabrikam Drone Delivery app we  wanted to streamline and align a bit the instructions following same designs/styling we started using from other RIs 

## WHAT Changed?

- use fixed resource group names reducing number of env vars
- more meaningful user managed identity names
- reduce number of variables in ARM templates favoring in-line-reading.

## HOW to test?

Follow the `README.md` steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,54 +26,38 @@ The Drone Delivery application is a sample application that consists of several
...
+WORKFLOW_PRINCIPAL_ID=$(az identity show -g rg-shipping-dronedelivery -n uid-workflow --query principalId -o tsv) && \
+PACKAGE_ID_PRINCIPAL_ID=$(az identity show -g rg-shipping-dronedelivery -n uid-package --query principalId -o tsv) && \
+INGESTION_ID_PRINCIPAL_ID=$(az identity show -g rg-shipping-dronedelivery -n uid-ingestion --query principalId -o tsv)
 ```
 
 ### Deploy the workload
 
 ```bash
-az deployment group create -f workload-stamp.json -g $ACR_RESOURCE_GROUP -p droneSchedulerPrincipalId=$DRONESCHEDULER_PRINCIPAL_ID \
+az deployment group create -f workload-stamp.json -g rg-shipping-dronedelivery-acr -p droneSchedulerPrincipalId=$DRONESCHEDULER_PRINCIPAL_ID \
```
Comment (ckittel) [inline README.md:50]: Q: doesn't this still then deploy the workload resources to `rg-shipping-dronedelivery-acr` instead of `rg-shipping-dronedelivery`?

**Review by ckittel — CHANGES_REQUESTED** [review]
Just one validation before I punch Approve on this.  Structurally, the changes all look good, just want to validate the one item here.

File: README.md (hunk 1 above)
Comment (ferantivero) [reply README.md:50]: yeah, exactly we have another coming PR for that. Please let me share that with you from [here](https://github.com/mspnp/fabrikam-dronedelivery-workload/pull/19).

Comment (ckittel) [reply README.md:50]: Sounds good!

**Review by ckittel — APPROVED** [review]
👍 +1"

**Pull request merged**